Ensure no sudo wrapper given may easily result in an error with no permission
to set TERMINFO. Normally disabled by "no-sudo" but depending on load ordering
or user configs overwriting that without disabling it as well, it could give
poor surprises.

Not needed on Gentoo given have a kitty-terminfo package installed in standard
paths without needing TERMINFO set.
--- a/shell-integration/bash/kitty.bash
+++ b/shell-integration/bash/kitty.bash
@@ -215,4 +214,0 @@
-    if [[ "${_ksi_prompt[sudo]}" == "y" ]]; then
-        # Ensure terminfo is available in sudo
-        [[ -n "$TERMINFO" ]] && builtin alias sudo="sudo TERMINFO=\"$TERMINFO\""
-    fi
--- a/shell-integration/zsh/kitty-integration
+++ b/shell-integration/zsh/kitty-integration
@@ -391,4 +390,0 @@
-    if (( ! opt[(Ie)no-sudo] )); then
-        # Ensure terminfo is available in sudo
-        [[ -n "$TERMINFO" ]] && builtin alias sudo="sudo TERMINFO=\"$TERMINFO\""
-    fi
